pallets/parachain-staking/src/lib.rs

```diff
@@ -1205,7 +1276,16 @@ pub mod pallet {
 				let round_to_payout = next - duration;
 				let total = <Points<T>>::get(round_to_payout);
 				let total_staked = <Staked<T>>::get(round_to_payout);
-				let issuance = Self::compute_issuance(total_staked);
+				let mut issuance = Self::compute_issuance(total_staked);
+				// reserve portion of issuance for parachain bond account
+				let bond_config = <ParachainBondInfo<T>>::get();
+				let parachain_bond_reserve = bond_config.percent * issuance;
+				if let Ok(imb) =
+					T::Currency::deposit_into_existing(&bond_config.account, parachain_bond_reserve)
+				{
+					// update round issuance iff transfer succeeds
+					issuance -= imb.peek();
+				}
```

Comment:
Would it be useful to have a special event emitted here?

Reply:
Yes, I'll add one -- will make the event log more transparent by including this.